src/management/JobsManager.js

```diff
@@ -156,6 +157,10 @@ JobsManager.prototype.get = function(params, cb) {
  * @return  {Promise|undefined}
  */
 JobsManager.prototype.importUsers = function(data, cb) {
+  console.warn(
+    'importUsers has been deprecated as it was inconsistent with our api. Please use importUsersJob which includes response data directly'
```

Comment:
Suggestion to improve a bit the message

```suggestion
    '"importUsers" has been deprecated as it was inconsistent with the API. Please, use "importUsersJob" which returns the response data directly.'
```